nem-openapi/spec/schemas/AllNodesDTO.yml

```diff
@@ -10,19 +10,19 @@ properties:
         type: array
```

Comment:
i think we've been using yaml extension everywhere, so might want to rename these (separate change)